feature-libs/order/components/order-history/order-history.component.spec.ts

```diff
@@ -220,17 +221,46 @@ describe('OrderHistoryComponent', () => {
     );
 
     expect(elements.length).toEqual(2);
+    expect(component.sortType).toEqual('byDate');
+  });
+  it('should not have sortType if no orders and pagination are provided', () => {
+    let orders: OrderHistoryList | undefined;
+
+    mockOrderHistoryList$.next(undefined);
+
+    component.orders$
+      .subscribe((value) => {
+        orders = value;
+      })
+      .unsubscribe();
+
+    expect(orders).toEqual(undefined);
+
+    expect(component.sortType).toBe(undefined);
```

Comment:
Are those changes related to this PR?

Reply:
Effort done by @mszczepaniak to bump unit test coverage above defined threshold for pipeline